nameko_grpc/channel.py

```diff
@@ -119,6 +119,8 @@ def __init__(self, host, port, ssl, spawn_thread, handle_request):
         self.handle_request = handle_request
 
         self.connections = queue.Queue()
+        self.is_accepting = False
```

Comment:
These were missing from the init. There's a few more cases of this but not fixing everything.